anda/lib/astal/ags/ags.spec

```
@@ -0,0 +1,86 @@
+# Generated by go2rpm 1.15.0
+%bcond check 1
+%bcond bootstrap 0
+
+%if %{with bootstrap}
+%global debug_package %{nil}
+%endif
+
+%if %{with bootstrap}
+%global __requires_exclude %{?__requires_exclude:%{__requires_exclude}|}^golang\\(.*\\)$
+%endif
+
+# https://github.com/Aylur/ags
+%global goipath         github.com/Aylur/ags
+Version:                2.3.0
+
+%gometa -f
+
+%global common_description %{expand:
+Scaffolding CLI for Astal+TypeScript.}
+
+%global golicenses      LICENSE
+%global godocs          docs README.md
+
+Name:           ags
+Release:        %autorelease
+Summary:        Scaffolding CLI for Astal+TypeScript
+
+License:        GPL-3.0-only
+URL:            %{gourl}
+Source:         %{gosource}
+Provides:       golang-github-aylur-ags = %version-%release
+Packager:       madonuko <mado@fyralabs.com>
+
+BuildDepends:  anda-srpm-macros
```

```suggestion
BuildRequires:  anda-srpm-macros

```
i am fucking cooked why is my brain thinking about deb packaging

---

LMAO